src/components/Navigation.jsx

```diff
@@ -4,39 +4,79 @@ import Paper from 'material-ui/Paper'
 import { List, ListItem } from 'material-ui/List'
 import Divider from 'material-ui/Divider'
 import { withRouter } from 'react-router'
-import { dataTest, camelCase } from '../lib/attributes'
+import { dataTest } from '../lib/attributes'
+import { FlatButton } from 'material-ui'
+import { StyleSheet, css } from 'aphrodite'
+
+const styles = StyleSheet.create({
+  sideBarWithMenu: {
+    width: 256,
+    height: '100%',
+    writingMode: 'hoizontal-lr',
+  },
+  sideBarWithoutMenu: {
+    writingMode: 'vertical-rl',
+    padding: '5px',
+    paddingTop: '20px'
+  }
+})
 
 const Navigation = function Navigation(props) {
   const { sections, switchListItem } = props
 
-  return (
-    <Paper
-      rounded={false}
-      zDepth={2}
-      style={{
-        height: '100%'
-      }}
-    >
-      <List>
-        {sections.map((section) => (
-          <ListItem
-            {...dataTest(camelCase(`nav ${section.path}`))}
-            key={section.name}
-            primaryText={section.name}
-            onTouchTap={() => props.router.push(section.url)}
-          />
-        ))}
-        <Divider />
-        {switchListItem}
-      </List>
-    </Paper>
-  )
+  if (props.showMenu) {
+    return (
+      <div className={css(styles.sideBarWithMenu)}>
+        <Paper
+          rounded={false}
+          zDepth={2}
+          style={{
+            height: '100%'
+          }}
+        >
+          <div style={{ display: 'flex', justifyContent: 'flex-end' }}>
+            <FlatButton
+              label={'Close Menu'}
+              onClick={props.onToggleMenu}
+            />
+          </div>
+
+          <List>
+            {sections.map(section => (
+              <ListItem
+                {...dataTest(camelCase(`nav ${section.path}`))}
```

Comment:
I believe `lodash` needs to be imported and `camelCase` needs to be invoked from the import for this to work.